flake.nix

```
@@ -0,0 +1,45 @@
+{
+  inputs = {
+    opam-nix.url = "github:tweag/opam-nix";
+    flake-utils.url = "github:numtide/flake-utils";
+    nixpkgs.follows = "opam-nix/nixpkgs";
+    cpspg.url = "github:Brychlikov/cpspg";
```

adampsz/cpspg#1 was merged, so this can be replaced with `cpspg.url = "github:adampsz/cpspg"`

---

Github has this cool feature for suggestions (works with multiline comments too):
```suggestion
    cpspg.url = "github:adampsz/cpspg";
```